python-mini-projects/projects/cli_proxy_tester/proxytest.py

```diff
@@ -0,0 +1,119 @@
+import logging
+from json.decoder import JSONDecodeError
+from pathlib import Path
+
+import pandas as pd
+import requests
+from requests.exceptions import ProxyError
+
+logging.basicConfig(level=logging.INFO)
+
+
+def add_proxies_to_file(csv_path: str, proxies: list):
+    '''This function will add one or multiple proxies to the CSV file.'''
+
+    if not csv_path.exists():
+        pr_file: pd.DataFrame = pd.DataFrame(
+            columns=['proxy_type', 'proxy_address', 'proxy_status'])
+        logging.info('New CSV file will be created')
+    else:
+        pr_file: pd.DataFrame = pd.read_csv(csv_path)
+        logging.info('Existing CSV file has been loaded')
+
+    for proxy in proxies:
+        if len(pr_file) == 0:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`.

 [Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)

Similar issue at line number 28.